src/pages/FindYourCommunity/YourLocalArea/Donate/Donate.tsx

```diff
@@ -131,9 +132,23 @@ const Donate: FC = () => {
     {
       title: 'Status',
       dataIndex: 'registered',
+      filters: [
```

Comment:
Any chance we can add text search to the school name also? Can reuse from the LA table this filter was already on